etc/mpv-powershell-completion.ps1

```diff
@@ -0,0 +1,287 @@
+#!/bin/pwsh
+
+# PowerShell command line completion for the mpv/mpv.net media player.
+
+# It can be installed by dot sourcing it in the PowerShell profile.
+
+#
+# This file is part of mpv.
+#
+# mpv is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# mpv is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+$Options = New-Object Collections.Generic.List[Object]
+$UpdateOptions = @('vo', 'ao', 'profile', 'hwdec', 'audio-device', 'scale', 'tscale',
```

Comment:
Any reason why only these options?

Reply:
These options have `=help` support, which the script uses to extract the choices by running the executable.

Reply:
But more options also support `help`...